queries.sql

```diff
@@ -1,3 +1,10 @@
 /*Queries that provide answers to the questions from all projects.*/
 
-SELECT * from animals WHERE name = 'Luna';
+SELECT * FROM animals WHERE name LIKE '%mon';
+SELECT name FROM animals WHERE date_of_birth BETWEEN '2016-01-01' AND '2019-12-31';
+SELECT name FROM animals WHERE neutered AND escape_attempts < 3;
+SELECT date_of_birth FROM animals WHERE name IN ('Agumon', 'Pikachu');
+SELECT name, escape_attempts FROM animals WHERE weight_kg > 10.5;
+SELECT * FROM animals WHERE neutered;
```

Comment:
- [ ] If the "neutered" column contains any other type of data, such as strings or integers, the query will not work as expected and may return an error or unexpected results.

- Therefore, to ensure correct behavior, it's best to explicitly compare the "neutered" column to a boolean value

`SELECT * FROM animals WHERE neutered = true`